build/parser.mjs

```diff
@@ -193,7 +193,7 @@ class Parser {
     this.addPatchlogs(result, data.patchlogs);
     this.addAdditionalWikiaData(result, category, data.wikia);
     this.detectPrime(result);
-    this.addVaultData(result, data.vaultData);
+    this.addVaultData(result, category, data.wikia, data.vaultData);
```

Comment:
you forgot to update the line 223 with the new arguments

Reply:
oh